assets/backup.sh

```
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# can be added as cronjob, for example as:
+#
+#   00 10 01 * * /path/to/backup.sh measurements.json
+#
+# to backup asset files periodically (here every month)
+
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)